test/helpers/test-data.js

```diff
@@ -1,17 +1,24 @@
 export const VALID_PARTICIPANT_ADDRESS = '0x000000000000000000000000000000000000dEaD'
 
-export const VALID_TASK = {
+export const VALID_TASK_WITH_PROVIDER_PROTOCOL = {
   cid: 'QmUuEoBdjC8D1PfWZCc7JCSK8nj7TV6HbXWDHYHzZHCVGS',
   providerAddress: '/dns4/production-ipfs-peer.pinata.cloud/tcp/3000/ws/p2p/Qma8ddFEQWEU8ijWvdxXm3nxU7oHsRtCykAaVz8WUYhiKn',
   protocol: 'bitswap'
 }
+Object.freeze(VALID_TASK_WITH_PROVIDER_PROTOCOL)
```

Comment:
We are not using this sample measurement in the old format. Should I completely delete the definition from our source code?

Reply:
👍